test/parallel/test-eslint-prefer-assert-methods.js

```diff
@@ -7,31 +7,46 @@ const rule = require('../../tools/eslint-rules/prefer-assert-methods');
 
 new RuleTester().run('prefer-assert-methods', rule, {
   valid: [
-    'assert.strictEqual(foo, bar)',
-    'assert(foo === bar && baz)'
+    'assert.strictEqual(foo, bar);',
+    'assert(foo === bar && baz);',
+    'assert.notStrictEqual(foo, bar);',
+    'assert(foo !== bar && baz);',
+    'assert.equal(foo, bar);',
+    'assert(foo == bar && baz);',
+    'assert.notEqual(foo, bar);',
+    'assert(foo != bar && baz);',
+    'assert.ok(foo);',
+    'assert.ok(foo != bar);',
```

Comment:
I could be wrong but I think the idea was that `assert(foo != bar)` should yield the same as `assert.ok(foo != bar)`. Since the former is an alias for the latter.

(I realize that might be modifying the current rule and is somewhat outside of the scope of the original work.)

Reply:
That would indeed be nice but I guess it is best to keep that for a separate PR and I am actually about to improve the assert message for cases like that in #17581

Reply:
+1